lib/tasks/krikri.rake

```diff
@@ -8,38 +10,73 @@ require 'krikri/index_service'
 
 namespace :krikri do
 
-  # Tasks will execute in spec/internal unless directory is changed
-  desc 'Index a valid sample aggregation'
-  task :index_sample_aggregation do
-    agg = build(:aggregation)
+  # Note: The content of the sample original record does not actually
```

Comment:
While we're in the this tasks file, maybe we should consider namespacing these all under something like `krikri:index`, or `krikri:samples`?